AppKit/CPSplitView.j

```diff
@@ -1205,6 +1186,32 @@ The sum of the views and the sum of the dividers should be equal to the size of
 
 @end
 
+@implementation CPSplitView (CPTrackingArea)
+
+- (void)updateTrackingAreas
+{
+    var myTrackingAreas = [self trackingAreas];
```

Comment:
**Please** don't use "my" as a prefix for variable names. `areas` or even `trackingAreas` is fine.